age.html

```diff
@@ -21,25 +20,32 @@
         }
         </style>
 </head>
+
 <body>
-    <div id="hero">
-        <div id="box">
-            <div>
-                <p>Please enter your date of birth:</p>
-                <form id="age-form">
-                    <input id="input" required type="number" id="day" name="day" placeholder="Day" size="4">
-                    <input id="input" required type="number" id="month" name="month" placeholder="Month" size="4">
-                    <input id="input" required type="number" id="year" name="year" placeholder="Year" size="4">
-                    <br>
-                    <div id="submit">
-                        <input id="submit" type="submit" value="Submit">
+    <div class="container">
+        <!-- https://howtogetonline.com/how-to-embed-a-hero-background-video.php -->
+          <section id="hero">
+                <video autoplay loop muted width="100%" id="bgvideo">
+                    <source src="./assets/video/loadingscreen.mp4" type="video/mp4">
+                </video>
+
+                <div id="box">
+                    <div>
+                        <p>Please enter your date of birth:</p>
+                        <form id="age-form">
+                            <input id="input" required type="number" id="day" name="day" placeholder="Day" size="4">
+                            <input id="input" required type="number" id="month" name="month" placeholder="Month" size="4">
+                            <input id="input" required type="number" id="year" name="year" placeholder="Year" size="4">
+                            <br>
+                            <div id="submit-button">
```

Comment:
useless div remove it. Also the same id for input and div. Which caused side effect. 3 boxes.
NB remove only div. do not touch input )